imap_processing/cdf/tests/test_imap_cdf_manager.py

```diff
@@ -24,7 +26,8 @@ def test_add_instrument_global_attrs():
 
 def testing_source_dir():
     # Create an ImapCdfAttributes object
-    imap_cdf_manager = ImapCdfAttributes("tests")
+    imap_cdf_manager = ImapCdfAttributes()
+    imap_cdf_manager.source_dir = Path(__file__).parent.parent / "tests"
```

Comment:
You actually want to leave this step like you had it before - because you're testing that passing "tests" to the init results in source_dir getting set properly.